packages/utils/src/common.cairo

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.15.0 (utils/common.cairo)
+
+use core::traits::PartialOrd;
+
+pub impl Felt252PartialOrd of PartialOrd<felt252> {
```

Comment:
Maybe `Felt252AsIntPartialOrd` is a better name? Since order doesn't really make much sense for felts as a concept, maybe the name can help to clarify the intention.

Reply:
i removed this from the public API, since I don't feel comfortable providing the implementation ourselves (I'm using it locally in hashes though)